docs/cli.rst

```diff
@@ -48,6 +51,12 @@ Or with a filename::
     export FLASK_APP=/path/to/hello.py
     flask run
 
+If you use Flask 0.13 or later you can also set the environment variables
+in a ``.env`` file.  For instance like this::
+
+    FLASK_APP=/path/to/hello.py
+    FLASK_DEBUG=1
+
```

Comment:
Let me summarize. It will be very tempting to use the `.env` just so that you don't have to think about `FLASK_APP` anymore. As such, one might be inclined to commit this `.env` file to git, so that you don't have to make one in every machine the app is installed. Since many other frameworks use a `.env` file as a place to store secrets, this presents a conflict, in my opinion. We would be using the .env file in a different way that what I consider the norm.

As another issue, having ` .env` file that is not imported when you run the app in production via gunicorn, etc. is also a problem, if you use it to store other settings that are needed by the application, besides `FLASK_APP` and `FLASK_DEBUG`.

For `FLASK_APP`, I would rather implement some sort of default, so that you don't have to rely on configuration. For example, I would be okay with `FLASK_APP` defaulting to `wsgi:application`. You then write a two line `wsgi.py`, safely commit it to git and be done with it.